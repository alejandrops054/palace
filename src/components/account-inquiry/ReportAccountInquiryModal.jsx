/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos 
*/

import React from 'react'
import {ModalReact} from '../index'
import ReportAccountInquiry from './ReportAccountInquiry'

function ReportAccountInquiryModal() {
    return (
        <>
         
        </>
    )
}

export default ReportAccountInquiryModal
