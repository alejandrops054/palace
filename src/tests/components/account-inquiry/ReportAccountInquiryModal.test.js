/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react'
import {shallow} from 'enzyme'

import ReportAccountInquiryModal from '../../../components/account-inquiry/ReportAccountInquiryModal'

describe('Pruebas del componente ReportAccountInquiryMI ', () => {
    test('Creacion de ReportAccountInquiryMI', () => {
        
        const wrapper = shallow(<ReportAccountInquiryModal/>)

        expect(wrapper).toMatchSnapshot();
    })
    
})