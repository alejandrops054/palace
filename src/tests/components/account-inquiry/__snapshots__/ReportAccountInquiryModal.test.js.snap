// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente ReportAccountInquiryMI  Creacion de ReportAccountInquiryMI 1`] = `<Fragment />`;
